.github/workflows/apidocs.yml

```
@@ -0,0 +1,26 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install apidoc
+        run: npm install --no-save apidoc
+
+      - name: Generate docs
+        run: npx apidoc -c apidoc.json
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)